src/pages/authentication-pages/signup/Signup.jsx

```diff
@@ -22,10 +22,7 @@ function Signup() {
     (async () => {
       try {
         const response = await axios.post("/api/auth/signup", userData);
-        localStorage.setItem(
-          "token",
-          JSON.stringify(response.data.encodedToken)
-        );
+        localStorage.setItem("token", response.data.encodedToken);
         navigate("/");
       } catch (error) {
         console.log(error);
```

Comment:
remove console log

Reply:
I have kept that to see errors , since I have not added , toast yet